src/redux/mission/missionSlice.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ] [REQUIRED] Kindly enable eslint. Disabling eslint will not allow it to check for the Javascript code standards. Please, ensure to fix the linter errors manually.